src/config/konnectors.json

```diff
@@ -1805,12 +1805,37 @@
       "bankId": "95"
     }
   },
+  {
+    "slug": "inulogic",
+    "name": "Inulogic",
+    "editor": "Cozy Cloud",
+    "vendorLink": "https://www.inulogic.fr",
+    "categories": ["other"],
+    "fields": {
+      "login": {
+        "type": "email"
+      },
+      "password": {
+        "type": "password"
+      },
+      "advancedFields": {
+        "folderPath": {
+          "advanced": true,
+          "isRequired": false
+        }
+      }
+    },
+    "dataType": [
+      "bill"
+    ],
+    "source": "registry://inulogic/stable"
+  },
```

Comment:
The icon declaration is missing. Cf. https://github.com/cozy/cozy-collect/pull/793/files#diff-043553f81194d48179200230a869eff0R1781

Reply:
Oh, I was told this opposite [here](https://github.com/cozy/cozy-collect/pull/779#issuecomment-392604629). I don't manage this repo, so if it's not an oversight, it's okay with me.